dispatch/tests/apis.py

```diff
@@ -0,0 +1,103 @@
+from dispatch.vendor.apis import Facebook, FacebookAPIError
+from dispatch.tests.cases import DispatchAPITestCase
+from dispatch.tests.helpers import DispatchTestHelpers
+
+class FacebookTest(Facebook):
+
+    API_ROOT = 'https://graph.facebook.com/v2.8'
+
+    def __init__(self, access_token=None):
+
+        self.access_token = access_token
+
+    def get_access_token(self, params):
+
```

Comment:
You can make this even simpler and just use an empty method:

```py
def get_access_token(self, params):
    pass
```

Same goes for the `__init__` method above